FinSearchUnified/FinSearchUnified.php

```diff
@@ -56,6 +63,15 @@ public function install(InstallContext $context)
     {
         $context->scheduleClearCache([InstallContext::CACHE_TAG_THEME]);
         parent::install($context);
+
+        // Manually load the config xml due to Shopware 5.2.0 compatibility
+        $config = $this->loadConfig();
```

Comment:
Also do this on update.

Reply:
Probably makes sens to put this into an own method. E.g. `initializeConfiguration` or something like that.